### PR TITLE
#3932, remove setSelectionRange for number inputs

### PR DESCRIPTION
**Pull request by DaveBagler**
See this issue for more information: https://github.com/formio/formio.js/issues/3932

### CODE REVIEW

File: src/components/number/Number.js [hunk 1]
```diff
@@ -178,7 +178,6 @@ export default class NumberComponent extends Input {
     const input = this.refs.input[0];
     if (input) {
       super.focus.call(this);
-      input.setSelectionRange(0, input.value.length);
```
Comment (travist) [inline src/components/number/Number.js:181]: We would prefer that you just add a try {} catch statement around this so it is still here but does not crash IE11.

**Review by travist — CHANGES_REQUESTED** [review]

File: src/components/number/Number.js (hunk 1 above)
Comment (DaveBagler) [reply src/components/number/Number.js:181]: I can do that, but just to be clear, the issue isn't IE11 crashing. Older browsers seemed to run the method without issue, it's the newer browsers that are throwing a DOMException.

Comment (DaveBagler) [reply src/components/number/Number.js:181]: See update in new commit.

**Comment by travist** [comment]
I believe this PR will solve this problem. https://github.com/formio/formio.js/pull/3966

**Comment by DaveBagler** [comment]
@travist, PR #3966 updates component.js and day.js. It doesn't remove the setSelectionRange() call from number.js.

**Comment by DaveBagler** [comment]
@travist, ping on this PR.

**Pull request closed without merging**